yang/ietf-energy-saving-mgt.yang

```diff
@@ -196,34 +198,43 @@ module ietf-energy-saving-mgt {
        will not be lost.";
   }
 
-  identity energy-saving-oper-state {
+  identity energy-saving-power-state {
     description
-      "Base identity for energy-saving-oper-state.";
+      "Base identity for energy-saving-power-state.";
+	reference "RFC7326: Energy Management Framework";
   }
 
-  identity energy-saving {
-    base energy-saving-oper-state;
+  identity off-state {
+    base energy-saving-power-state;
     description
-      "Identity to indicate support of energy-saving mode.";
+      "Indicates that the device typically 
+	   requires a complete boot when awakened.";
+	reference "RFC7326: Energy Management Framework";
   }
 
-  identity unsupported {
-    base energy-saving-oper-state;
+  identity sleep-state {
+    base energy-saving-power-state;
     description
-      "Indicates that the device does not support energy-saving
-       mode or does not have enough resources.";
+      "Identity to indicate components with energy management support 
+	   are not functional but immediately available such as wake up 
```

Comment:
```suggestion
	   is not functional but immediately available such as wake up 
```

Reply:
I think we should use "are" instead of 'is' since 'components' is plural.

Reply:
As currently written, the identity applies only to a device while I think we should not have that restriction to cover devices, entities, components, etc.  The full description is to be updated, I'm afraid. Thanks.

Reply:
OK, I agree with your proposed change, just point out small nit. It is okay now, I can accept your commit suggestion.